src/UI/Component/Chart/Factory.php

```diff
@@ -28,4 +28,26 @@
 	 * @return  \ILIAS\UI\Component\Chart\ScaleBar
 	 */
 	public function scaleBar(array $items);
+
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     Speedos are used to display a progress, in the form of an speedometer.
+     *     E.g. they can be used to inform about a progress in a course or other
+     *     learning objectives.
+     *   composition: >
```

Comment:
The user's achievement is represented by a bar in one color and indicates the specific achievement. The maximum achievement is represented by an other color. Achievements can be in given as points or percentages.